node/src/chain_spec.rs

```diff
@@ -485,7 +485,7 @@ fn create_testnet_claims() -> Vec<(EthereumAddress, Balance)> {
 		// Test seed: "image stomach entry drink rice hen abstract moment nature broken gadget flash"
 		// private key (m/44'/60'/0'/0/0) : 0xdd75dd5f4a9e964d1c4cc929768947859a98ae2c08100744878a4b6b6d853cc0
 		EthereumAddress(hex!["8202C0aF5962B750123CE1A9B12e1C30A4973557"]),
```

Comment:
Will this actually work? (upper/lowercase)

Reply:
Tests pass. AFAIK same wallets use capitalization to compute a checksum and validate an address. Hex conversion algorithms should be clever enough to handle both lower and uppercase input. ('a' and 'A' are the same in hex).

Reply:
We should probably test this with metamask as I know that there were some problems with this and also we're lowercasing addresses in UI. I'll do that when I have time.

Reply:
I've generated all signatures using mycrypto + metamask. It's also easy to convert 3 public keys to lowercase and update the source code.

Reply:
It looks like it works anyway so all good.